src/core/opamStd.ml

```diff
@@ -903,19 +903,18 @@ module OpamSys = struct
       let split = if clean then OpamString.split else OpamString.split_delim in
       split path path_sep
 
-  let with_process_in cmd args f =
+  let process_in cmd args =
     if Sys.win32 then
       assert false;
```

Comment:
Should we keep that assert false or have a nicer error ?

Reply:
the assert false should stay since this function is not supposed to be called on windows and that it is not exported.